gsa/src/gmp/models/task.js

```diff
@@ -86,6 +87,14 @@ export function parse_yes(value) {
   return value === 'yes' ? YES_VALUE : NO_VALUE;
 }
 
+function parseIntoArray(obj) {
+  if (!isArray(obj)) {
+    const array = Array(obj);
```

Comment:
```suggestion
    return [obj];
```
this function can also be shortened to
```javascript
const parseIntoArray = value => isArray(value) ? value : [value];
```

Reply:
This looks much cleaner. Thanks!